content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -230,30 +226,75 @@ clusters, zones, or regions.
 This model affords the following benefits:
 
 - Improved availability: If a control plane becomes unavailable, the scope of
-  the outage is limited to only that control plane.
+  the outage is limited to only workloads in clusters managed by that control plane.
+
+- Improved configuration isolation: Configuration changes in one control plane affects only
+the workloads in clusters managed by this control plane.
+
+### Control Plane Per Cluster
+
+For maximum availability and configuration isolation you can deploy a control plane in each cluster. Each cluster's control plane
+manages only the workloads in its own cluster. If the control plane is unavailable (due to upgrade or
+temporary outage), the workloads will continue to function with the existing sidecar configuration and
+will re-attempt to connect to the in-cluster control plane periodically.
+
+This model affords the following benefits:
+
+- Highest level of availability: If a control plane becomes unavailable, the scope of
+  the outage is limited to only that cluster and the workloads within it.
+
+- Maximum configuration isolation: You can make configuration changes in one cluster without impacting
+  workloads deployed in any other cluster.
+
+### Availability vs Config Isolation
+
+The following list ranks control plane deployment examples by availability and configuration isolation:
+
+- One control plane per region or a single global control plane (**lowest availability**, **lowest isolation**)
+- Multiple control planes per region
+- One control plane per cluster (**highest availability**, **highest isolation**)
+
+## Mesh models
+
+Istio supports having all of your services in a
+{{< gloss "service mesh" >}}single mesh{{< /gloss >}}, or federating multiple meshes
+together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. A single mesh can span
+[one or more clusters](#cluster-models) and [one or more networks](#network-models).
+All workloads in a mesh share a common certificate authority. You can choose to use a single shared
+  trust domain or a separate trust domain for each cluster.
+If you use a single shared trust domain, a service account in a given namespace is globally unique - i.e. if
+two clusters `cluster-1` and `cluster-2` have the same service account `foo` in namespace `ns`, they will be
+treated the same for purposes of security. With a trust domain for each cluster, there are no
+restrictions on the uniqueness of service accounts, and namespaces. You can apply enforcement
+policies that allow or deny all traffic from one cluster (irrespective of the namespace/service) to another.
```

Comment:
@rshriram @frankbu what are the examples of enforcement policies here?  Istio RBAC or network policies?  I assume Istio RBAC doesn't work across clusters when there are multiple trust domains.

Reply:
as i mentioned earlier, it does and we use it in production.

Reply:
Can we clarify it is Istio authZ policy (vs a general enforcement policies) if that is what you are referring to here?

Note: we don't have examples for this at istio.io or in our github.  So this could be perceived as unsupported.

Reply:
done